### DISCUSSION TITLE
Compatibility

### DISCUSSION THREAD

**Discussion by lakerdoc** [Quantum-Manager/tracker#68] [General]
Hi
I like to know, is the qunatum Manager compatible with the Quix and/or the SP Page Builder?
If yes, which of them can you suggest?

**Comment by dmitriitux** [comment]
Plugin for integration with sp builder is, just recently released a fix for it.

**Reply by lakerdoc** [reply]
Tanks a lot

**Reply by starshiphamburger** [reply]
Hey guys, I accidentally posted a new thread that Lakerdoc posted. I cannot seem to see the integration in SP page builder when using JCE. However, this is fine in normal joomla articles. Any updates ? Appreciate any feedback

**Reply by dmitriitux** [reply]
> Hey guys, I accidentally posted a new thread that Lakerdoc posted. I cannot seem to see the integration in SP page builder when using JCE. However, this is fine in normal joomla articles. Any updates ? Appreciate any feedback

i didn't override the file manager in jce

**Reply by starshiphamburger** [reply]
> > Hey guys, I accidentally posted a new thread that Lakerdoc posted. I cannot seem to see the integration in SP page builder when using JCE. However, this is fine in normal joomla articles. Any updates ? Appreciate any feedback
> 
> i didn't override the file manager in jce

Hi dmitriitux, I see. Is there any possibility of updating Quantum so that it supports JCE ? We would very much appreciate it. I'm sure also the whole JED family would love this. Thanks to advise.

**Reply by dmitriitux** [reply]
in theory, a plugin can intercept. I'll see, probably in a month

**Reply by starshiphamburger** [reply]
> in theory, a plugin can intercept. I'll see, probably in a month

Thanks dmitriitux, very much appreciate it. Looking forward to an update. I already left a 5 star on JED. Thanks for the amazing work man.
